ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts

```diff
@@ -3,7 +3,10 @@ import { MhrCompVal, MhrSectVal } from '@/composables/mhrRegistration/enums'
 export const useMhrValidations = (validationState: any) => {
   /** Set specified flag */
   const setValidation = (section: MhrSectVal, component: MhrCompVal, isValid: boolean): void => {
-    validationState[section].value[component] = isValid
+    // Only sets specified flag if the section and component are part of the validation model
+    if (validationState[section]?.value[component] !== undefined) {
+      validationState[section].value[component] = isValid
+    }
```

Comment:
Great addition here, it was too forgiving previously and by allowing the creation of new properties, it was prone to error. Good check 👍